src/views/virtualmachines/details/tabs/configuration/storage/components/tables/disk/DiskRowActions.tsx

```diff
@@ -103,7 +101,13 @@ const DiskRowActions: FC<DiskRowActionsProps> = ({
           submitBtnVariant={ButtonVariant.danger}
         />
       ) : (
-        <DeleteDiskModal isOpen={isOpen} onClose={onClose} vm={vm} volume={volume} />
+        <DeleteDiskModal
+          isHotPluginVolume={isHotplug}
```

Comment:
if isHotplug come from helper why its prop and not directly consumed in deletemodal?